test/csuite/timestamp_abort/main.c

```diff
@@ -536,6 +552,9 @@ main(int argc, char *argv[])
 		 * kill the child, run recovery and make sure all items we wrote
 		 * exist after recovery runs.
 		 */
+		memset(&sa, 0, sizeof(sa));
+		sa.sa_handler = handler;
+		sigaction(SIGCHLD, &sa, NULL);
```

Comment:
Why did we need sigaction(2), wasn't signal(3) sufficient?

Reply:
Every man page for `signal` says that to be portable you should use `sigaction` instead.  Here's what it says on margay:
```
 The  behavior  of  signal() varies across UNIX versions, and has also varied historically across different versions of Linux.  Avoid its use: use sigaction(2) instead.  See Portability below.
```

Reply:
Huh... learn something every day...